core/templates/dev/head/pages/admin/admin.html

```diff
@@ -123,10 +123,10 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/expressions/ExpressionParserService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/domain/utilities/UrlInterpolationService.js"></script>
 
+    {% include 'pages/rich_text_components.html' %}
```

Comment:
It's not a page... maybe put this in components/ (or a subfolder of that) instead?

Reply:
Done.